planet/src/test/java/com/google/planet/ItineraryGeneratorTest.java

```diff
@@ -44,20 +44,22 @@ public void setUp() {
     public void scheduleItineraryInOrder() {
```

Comment:
maybe add comments for each test that describes what you are testing

Reply:
Okay!